docs/tutorials/HD143006_Part_2.py

```diff
@@ -0,0 +1,481 @@
+
+# +
+# # %tensorboard --logdir {logs_base_dir}
+## uncomment the above line when running to view TensorBoard
+# -
+
+# ## Training and Imaging Part 2: Cross Validation
+#
+# Now we will move into the realm of cross validation. Cross validation is a technique used to assess model validity. This is completed by storing one chunk of a dataset as the test dataset and using the remaining data to train the model. Once the model is trained, it is used to predict the values of the data in the test dataset. These predicted values are compared to the values from the test dataset, producing a cross validation score. The advantage of cross validation is that it allows one dataset to be used to train the model multiple times since it can take different chunks out for the test dataset. For more information see the [Cross Validation tutorial](crossvalidation.html).
+#
+# Just like in the previous section we will be viewing our results in TensorBoard, with the addition of the cross validation score log.
+
+# Cross validation requires a `test` function (to determine the cross validation score) and a `cross_validate` function (to utilize cross validation with the previous `train` function). We implement these below.
+
+
+def test(model, dataset):
+    model.eval()
+    vis = model.forward()
+    loss = losses.nll_gridded(
+        vis, dataset
+    )  # calculates the loss function that goes to make up the cross validation score
+    return loss.item()
+
+
+def cross_validate(model, config, k_fold_datasets, MODEL_PATH, writer=None):
+    test_scores = []
+
+    for k_fold, (train_dset, test_dset) in enumerate(k_fold_datasets):
+
+        # reset model
+        model.load_state_dict(torch.load(MODEL_PATH))
+
+        # create a new optimizer for this k_fold
+        optimizer = torch.optim.Adam(model.parameters(), lr=config["lr"])
+
+        # train for a while
+        train(model, train_dset, optimizer, config, writer=writer)
+        # evaluate the test metric
+        test_scores.append(test(model, test_dset))
+
+    # aggregate all test scores and sum to evaluate cross val metric
+    test_score = np.sum(np.array(test_scores))
+
+    # adds cross validation score
+    if writer is not None:
+        writer.add_scalar("Cross Validation", test_score)
+
+    return test_score
+
+
+# Now, with our functions defined, we need to do the critical part of dividing our dataset into training and test datasets. There are many ways of going about this but here we are splitting it radially and azimuthally and removing chunks. This is visualized in the [Cross Validation tutorial](crossvalidation.html).
```

Comment:
You already cite the cross validation tutorial so maybe this isn't necessary, but maybe mention that MPoL's Dartboard is an easy built-in way to get the polar coord grid, and maybe very briefly mention why we are choosing to do it this way.